.github/workflows/solc_version.yml

```
@@ -81,48 +81,50 @@ jobs:
               ../=$(pwd)/contracts/ \
               @=node_modules/@ \
               solidity-bytes-utils/=node_modules/solidity-bytes-utils/ \
-              lsp0/=packages/LSP0ERC725Account/ \
-              lsp1/=packages/LSP1UniversalReceiver/ \
-              lsp1delegate/=packages/LSP1UniversalReceiverDelegate/ \
-              lsp2/=packages/LSP2ERC725YJSONSchema/ \
-              lsp3/=packages/LSP3ProfileMetadata/ \
-              lsp4/=packages/LSP4DigitalAssetMetadata/ \
-              lsp5/=packages/LSP5ReceivedAssets/ \
-              lsp6/=packages/LSP6KeyManager/ \
-              lsp7/=packages/LSP7DigitalAsset/ \
-              lsp8/=packages/LSP8IdentifiableDigitalAsset/ \
-              lsp9/=packages/LSP9Vault/ \
-              lsp10/=packages/LSP10ReceivedVaults/ \
-              lsp12/=packages/LSP12IssuedAssets/ \
-              lsp14/=packages/LSP14Ownable2Step/ \
-              lsp16/=packages/LSP16UniversalFactory/ \
-              lsp17/=packages/LSP17Extensions/ \
-              lsp17contractextension/=packages/LSP17ContractExtension/ \
-              lsp20/=packages/LSP20CallVerification/ \
-              lsp25/=packages/LSP25ExecuteRelayCall/ \
-              universalprofile/=packages/UniversalProfile/
+              @lukso/lsp0-contracts/=packages/LSP0ERC725Account/ \
```

One thing that could be done to simplify the remappings is have the folders named the same way as the package names, and we would just have to add the following entry in the remapping:

```suggestion
              @=packages/@ \
```

This would make it easier to not forget to add these entries in the CI + Foundry remapping file. Although the CI would still catch that in its current state.

Wdyt @YamenMerhi @skimaharvey @b00ste @richtera ?